tests/layer_tests/tensorflow_lite_tests/test_tfl_Unary.py

```diff
@@ -58,12 +60,15 @@ class TestTFLiteUnaryLayerTest(TFLiteLayerTest):
     def make_model(self, params):
         assert len(set(params.keys()).intersection({'op_name', 'op_func', 'shape'})) == 3, \
             'Unexpected parameters for test: ' + ','.join(params.keys())
-        self.allowed_ops = [params['op_name']]
+        if params['op_name'] != '':
```

Comment:
Do I understand correctly that tflite model has some other operations than GELU? What are the other types of operations?

We have to check that tflite model contains GELU op. Otherwise, next `tensorflow` module upgrade may bring changes in tf.nn.gelu translation to tflite and we'll lose the code coverage here without knowing it.

Reply:
Fixed.